iOS/DuckDuckGo/Refactoring/Updated/UpdatedOmniBarViewController.swift

```diff
@@ -141,6 +125,50 @@ final class UpdatedOmniBarViewController: OmniBarViewController {
     override func preventShadowsOnBottom() {
         omniBarView.updateMaskLayer(maskTop: false)
     }
+
+    // MARK: - Private Helper Methods
+
```

Comment:
Small refactor to make the textFieldShouldBeginEditing function better to read. No functional changes.